hedera-mirror-monitor/src/main/java/com/hedera/mirror/monitor/config/MonitorConfiguration.java

```
@@ -63,14 +63,14 @@
     Disposable publishSubscribe() {
         return Flux.<PublishRequest>generate(sink -> sink.next(transactionGenerator.next()))
                 .retry()
-                .filter(Objects::nonNull)
-                .doFinally(s -> log.warn("Stopped after {} signal", s))
                 .subscribeOn(Schedulers.single())
                 .parallel(publishProperties.getConnections())
                 .runOn(Schedulers.newParallel("publisher", publishProperties.getConnections()))
                 .map(transactionPublisher::publish)
-                .filter(Objects::nonNull)
-                .doOnError(t -> log.error("Error during publish/subscribe flow: ", t))
+                .sequential()
+                .onErrorContinue(PublishException.class, (t, r) -> {})
+                .onErrorContinue((t, r) -> log.error("Unknown error during publish/subscribe flow: {}", t))
```

- from IDEA hint: should remove `{}` in the message string
- the second `onErrorContinue` will swallow all `Throwable` other than `PublishException` which is taken care of in the first `onErrorContinue`. Is it intended? I saw in `PublishMetrics.java`, for `LocalValidationException`, the error message "Local error. Halting thread" hints something different.

---

Removed `{}`. Log4j2 allows it either way and will implicitly print stacktrace if you leave it off.

True, shouldn't continue if `LocalValidationException`. I've adjusted it to continue only for `PublishException` for now as that's the only thing I know for sure we should resume. All others will be logged and terminate the flux.